edx_exams/apps/lti/urls.py

```diff
@@ -8,6 +8,7 @@
 
 app_name = 'lti'
 urlpatterns = [
+    path('acs_endpoint/<int:lti_config_id>', views.acs_endpoint, name='acs_endpoint'),
```

Comment:
NIT: 'endpoint' seems kinda redundant to put in a url. Maybe just acs or assessment_control. I'd also recommend putting the config id first in this url since that id refers to the lti configuration and not the assessment. IMO putting it last implies it's an assessment/attempt id.

so` /lti/<lti_configuration>/acs` or something like that